tests/clusterclient_reconnect_async.c

```diff
@@ -74,6 +79,7 @@ void sendNextCommand(int fd, short kind, void *arg) {
         cluster_node *node = dictGetEntryVal(de);
         assert(node);
 
+        // coverity[tainted_scalar]
         int status = redisClusterAsyncCommandToNode(acc, node, replyCallback,
                                                     NULL, command);
```

Comment:
Here we silence the warning about `command` which is a printf-style format string taken unchecked from stdin.

I'm not extremely enthusiastic about tool-specific annotations in the code. If we have many tools, we will end up with many annotations to silence the warnings for each of them. I guess I can accept this one though if there is no other simple fix.

Reply:
After some reading it seems that the options would be to do some maneuvers like "hiding" it using `strdup`.
It might help by adding some length checks like `assert(len > 0);` and more, but maybe its not enough to make Coverity happy..
An alternative to having annotations is to mark it as Intentional in the tool.

Reply:
This comment is a way to mark it as intentional too, right? Marking it as intentional in the tool does not clutter our source code. But I don't care that much. We can merge.

Reply:
Good points. I just checked the coverity web frontend again, and it has "Action: Ignore" in the Triage section on the right - maybe we can use that in the future. But I'm not 100% sure if it will remember these settings if for example we move that file around, or move that code sample etc.

Afaics there is another option for coverity, to submit a "modeling" file as explained here: https://scan.coverity.com/tune - I'll look into that next time.